packages/web3-shared/src/hooks/useConstants.ts

```diff
@@ -0,0 +1,36 @@
+import { useMemo } from 'react'
+import { ChainId, useChainId } from './useChainId'
+
+type Primitive = string | number | boolean
```

Comment:
Should `symbol | bigint | undefined | null`?

```suggestion
type Primitive = string | number | boolean | symbol | bigint | undefined | null 
```